dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleMetadataConnector.java

```diff
@@ -198,22 +91,19 @@ public MessageTask(@Nonnull GroupTask... ts) {
     @Override
     protected Void doRun(TaskRunContext context, @Nonnull ByteSink sink, @Nonnull Handle handle)
         throws Exception {
-      int c = 1;
       LOG.error("All the select tasks failed:");
-      for (GroupTask task : tasks) {
+      for (int i = 1; i <= tasks.length; i++) {
         LOG.error(
-            "("
-                + c
-                + "): "
-                + task.getName()
-                + " : "
-                + ExceptionUtils.getRootCauseMessage(task.getException()));
-        c += 1;
+            String.format(
```

Comment:
I would suggest avoiding `String.format` in logger invocations. The logger has built-in parameter handling.

```
LOG.error("({}): {} : {}", i, tasks[i].getName(), ...);
```